examples_templates/example_scraper_2/humboldt_daily_scraper.py

```diff
@@ -0,0 +1,30 @@
+import os
+import sys
+
+from from_root import from_root
+
+p = from_root('CODE_OF_CONDUCT.md').parent
+sys.path.insert(1, str(p))
+
+from scrapers_library.data_portals.crimegraphics.crimegraphics_clery import crimegraphics_clery
```

Comment:
I think scrapers_library is scrapers now

Reply:
Yes, this was done in preparation for the next PR that is ready to go after this one, it will rename the `scrapers` directory to `scrapers_library` at the request of Josh. I just updated the reference preemptively so I wouldn't have to do it in the next PR and so I wouldn't forget to